packages/block-editor/src/components/inserter-draggable-blocks/index.js

```diff
@@ -2,15 +2,23 @@
  * WordPress dependencies
  */
 import { Draggable } from '@wordpress/components';
+
 /**
  * Internal dependencies
  */
 import BlockDraggableChip from '../block-draggable/draggable-chip';
 
-const InserterDraggableBlocks = ( { isEnabled, blocks, icon, children } ) => {
+const InserterDraggableBlocks = ( {
+	isEnabled,
+	blocks,
+	icon,
+	children,
+	clone = false,
```

Comment:
Do we really need the flag? Should we just always clone?

Reply:
Updated in db8cfe0 👍  we don't need a flag if we didn't have a use case for it.